bc_obps/service/form_builder_service.py

```diff
@@ -11,6 +17,16 @@ def str_to_camel_case(st: str) -> str:
     return output[0].lower() + output[1:]
 
 
+def get_fuel_list() -> List[str]:
+    """
+    Return the roles that are considered as authorized CAS users (excluding cas_pending).
+    """
+    try:
+        return list(FuelType.objects.all().values_list("name", flat=True))
+    except Exception:
+        return []
```

Comment:
What exception would we expect here? We should probably let our `@handle_http_errors()` decorator deal with it at the API level instead of swallowing whatever might have gone wrong

Reply:
This method should probably be part of the data access service, and could implement a caching mechanism (See RegulatedProductDataAccessService or NaicsCodeDataAccessService)